projects/flimnap/main.cc

```diff
@@ -89,6 +89,14 @@ namespace {
         return aout;
     }
 
+    constexpr float convert_adc_to_Nm(adcsample_t value, adcsample_t adc_zero, float magnitude) {
```

Comment:
Drop `convert_`, thats clear from the word `to`. I'd consider making it all lower case for consistency. `adc_to_nm` and add a comment about what it does if necessary.